packages/vue/src/components/confirm-sign-in.vue

```diff
@@ -1,66 +1,61 @@
 <script setup lang="ts">
+import { computed, toRefs, useAttrs } from 'vue';
 import {
   authenticatorTextUtil,
-  getActorState,
   getFormDataFromEvent,
-  SignInState,
   translate,
 } from '@aws-amplify/ui';
-import { computed, ComputedRef, useAttrs } from 'vue';
 
-import { useAuth, useAuthenticator } from '../composables/useAuth';
+import { useAuthenticator } from '../composables/useAuth';
+import { UseAuthenticator } from '../types';
```

Comment:
`UseAuthenticator` is `AuthenticatorServiceFacade` plus internal fields such as `QRFields`. React uses the same typing strategy: [L37-L46](https://github.com/aws-amplify/amplify-ui/blob/13d529369749d4771cfd3849b42b9f2c0fd4adfe/packages/react-core/src/Authenticator/hooks/useAuthenticator/types.ts#L37-L46).